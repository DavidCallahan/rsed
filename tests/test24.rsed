template='($user, $date)'r
$user='david'
$date='new'
print $template
print expand($template " again")
