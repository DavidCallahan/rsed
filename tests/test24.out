($user, $date)
(david, new) again
